doc/Language/statement-prefixes.pod6

```diff
@@ -80,16 +80,18 @@ don't care.
 
 =head2 X<C<quietly>|Syntax,quietly (statement prefix)>
 
-As a statement prefix, C<quietly> suppresses all warnings produced by the
-statement it precedes.
+As a prefix, C<quietly> suppresses all warnings produced by the
+block or statement it precedes (currently only run-time warnings).
 
```

Comment:
Suggestion:

``
As a prefix, C<quietly> suppresses all (run-time) warnings produced by the
block or statement it precedes.
``

Reply:
Done.